src/Gateio/Futures/API/Candle.jl

```diff
@@ -0,0 +1,117 @@
+module Candle
+
+export CandleQuery,
+    CandleData,
+    candle
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Gateio
+using CryptoAPIs: Maybe, APIsRequest
+
+@enum TimeInterval s10 m1 m5 m15 m30 h1 h4 h8 d1 d7 d30
+
+Base.@kwdef struct CandleQuery <: GateioPublicQuery
+    contract::String
+    from::Maybe{Int64} = nothing
+    to::Maybe{Int64} = nothing
```

Comment:
There should be a `DateTime` here

https://github.com/bhftbootcamp/CryptoAPIs.jl/blob/master/src/Gateio/Spot/API/Candle.jl#L17